src/util/Types.tsx

```diff
@@ -22,8 +22,58 @@ export type Member = {
   signInBlocked: boolean;
 };
 
+export type Report = {
+  id: Number;
+  member_id: Number;
+  reported_time: Date;
+  report_reason: string;
```

Comment:
If we're deciding on a default value for this when someone unregisters from an event (since there won't be text input) can we confirm it quickly with Matt?